astropy/units/tests/test_equivalencies.py

```diff
@@ -526,6 +526,17 @@ def test_brightness_temperature():
             u.Jy, equivalencies=u.brightness_temperature(omega_B, nu)))
 
 
+def test_thermodynamic_temperature():
+    nu = 143 * u.GHz
+    tb = 0.0026320518775281975 * u.K
+    np.testing.assert_almost_equal(
+        tb.value, (1 * u.MJy/u.sr).to_value(
+            u.K, equivalencies=u.thermodynamic_temperature(nu)))
+    np.testing.assert_almost_equal(
+        1.0, tb.to_value(
+            u.MJy / u.sr, equivalencies=u.thermodynamic_temperature(nu)))
```

Comment:
Just for completeness, also do a test with the temperature actually passed in.

Reply:
@mhvk , I am attempting to wrap this up but I wasn't involved in the original discussions. Could you please elaborate by "a test with  the temperature actually passed in"? Thanks!

Reply:
I think I addressed your comment via the `doctest` portion but I am not sure...

Reply:
The test was actually added (as a second test); since it didn't change the line I commented on, the comment didn't disappear.